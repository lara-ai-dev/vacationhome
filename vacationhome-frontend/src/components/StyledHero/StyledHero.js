import React from 'react';
import styled from 'styled-components';
import defaultImg from '../../images/apartment-1.jpeg';

//styled components for each individual room hero image
const StyledHero = styled.header`
    min-height: 60vh;
    background: url(${props => props.img? props.img : defaultImg}) center/cover no-repeat;
    display: flex;
    align-items: center;
    justify-content: center;
`

export default StyledHero;
