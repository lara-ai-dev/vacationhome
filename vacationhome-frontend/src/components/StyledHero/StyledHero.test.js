import { render } from '@testing-library/react'
import StyledHero from './StyledHero'

describe('StyledHero', () => {
    test('renders correctly', () => {
        const { container } = render(
            <StyledHero>
                <p>hello</p>
            </StyledHero>
        )

        expect(container).toMatchSnapshot()
    })
})
