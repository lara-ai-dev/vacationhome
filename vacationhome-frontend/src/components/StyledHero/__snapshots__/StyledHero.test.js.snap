// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledHero renders correctly 1`] = `
<div>
  <header
    class="sc-bdfBwQ bKhFOX"
  >
    <p>
      hello
    </p>
  </header>
</div>
`;
